crates/turborepo-lib/src/daemon/server.rs

```diff
@@ -281,9 +281,15 @@ impl<T: Watcher + Send + 'static> proto::turbod_server::Turbod for DaemonServer<
             )
             .await;
 
-        Ok(tonic::Response::new(proto::GetChangedOutputsResponse {
-            changed_output_globs: changed.into_iter().collect(),
-        }))
+        match changed {
+            Ok(changed) => Ok(tonic::Response::new(proto::GetChangedOutputsResponse {
+                changed_output_globs: changed.into_iter().collect(),
+            })),
+            Err(e) => {
+                error!("failed to watch flush directory: {:?}", e);
```

Comment:
Is this failing to register a watcher or not having the flush itself fail?

Reply:
Can be both so made the message more generic.